pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -1294,8 +1300,7 @@ public static int getKitCounter(@NonNull Connection conn, String collaboratorSam
      * and checking it against db if unique
      */
     public static String generateDdpLabelID() {
-        return generateDdpLabelID(15,
-                DSMConfig.getSqlFromConfig(ApplicationConfigConstants.GET_FOUND_IF_KIT_WITH_DDP_LABEL_ALREADY_EXISTS));
+        return generateDdpLabelID(15, GET_FOUND_IF_KIT_WITH_DDP_LABEL_ALREADY_EXISTS);
```

Comment:
Moved these two queries from conf file into class, it's a pattern we have been following for the last years to move queries into classes when we change the classes

Reply:
Yes, we started making them static as an overall rule a couple of years ago. To name one issue, having all of them in the config reduced code readability and made it hard to debug and track the impacts of our changes